rockspec/skywalking-nginx-lua-master-0.rockspec

```diff
@@ -0,0 +1,31 @@
+package = "skywalking-nginx-lua"
```

Comment:
What is this for? Provide one for dev version?

Reply:
only for developers. the developer will put the latest configuration here. 
when we release a new version, then we can copy it from this file directly.

Apache APISIX uses this way too:

https://github.com/apache/apisix/blob/master/rockspec/apisix-master-0.rockspec